docs/COMMAND_SURFACE.md

```
@@ -31,17 +31,25 @@
 4. `uc benchmark`
 - Runs benchmark matrix harness script.
 
-5. `uc session-key`
+5. `benchmarks/scripts/run_strict_supported_set_benchmarks.sh`
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 5; Style: 1/1/1

(MD029, ol-prefix)